{{cookiecutter.project_slug}}/.env.example

```diff
@@ -1,6 +1,5 @@
 DEBUG=1
 HOST=localhost
-PORT=8000
```

Comment:
you can't delete this, because it's used by `docker-compose.override.yml`
if it clashes with node, feel free to rename it in both places